FaceDetectionServer/moedx/object_detection/config/coco.data

```diff
@@ -0,0 +1,6 @@
+classes= 80
+train=data/coco/trainvalno5k.txt
+valid=data/coco/5k.txt
+names=data/coco.names
+backup=backup/
+eval=coco
```

Comment:
All code under the "config" directory is straight from https://github.com/cfotache/pytorch_objectdetecttrack.